include/esb.h

```diff
@@ -238,6 +245,15 @@ struct esb_config {
 				   *  will be acknowledged ignoring the noack
 				   *  field.
 				   */
+	bool use_fast_ramp_up; /**<  When this feature is enabled, radio TXEN and
+				 *  RXEN delays are reduced from 130 µs to 40 µs.
+				 *  The radio peripheral needs some time to start up
+				 *  analog components of the radio. On nRF51 and
+				 *  nRF24L series, a hard-coded 130 µs delay is
+				 *  implemented. If ESB connection is achieved only
+				 *  between nRF52 and/or nRF53 devices, this delay can
+				 *  be cut off to 40 µs.
```

Comment:
```suggestion
				 *  analog components of the radio. On the nRF51 and
				 *  nRF24L Series devices, a hard-coded 130 µs delay is
				 *  implemented. If ESB connection is achieved only
				 *  between nRF52 or nRF53 Series devices, this delay can
				 *  be reduced to 40 µs.
```

Reply:
Let's keep "end/or" to show that there are three possibilities: 52-52, 53-53, and 52-53.

Reply:
Done